www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+Below, you’ll find a chart that maps the “core” admin levels in India’s administrative hierarchy to current (and proposed) Who’s On First placetypes.
+
+
+<table>
+  <tr>
+   <td><strong>Indian Admin Level</strong>
+   </td>
+   <td><strong>WOF placetype</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>State</em>
+   </td>
+   <td><em>Region</em>
+   </td>
+   <td><em>In WOF pre-Karmashapes</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Union territory</em>
+   </td>
+   <td><em>Region</em>
+   </td>
+   <td><em>In WOF pre-Karmashapes</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>District</em>
+   </td>
+   <td><em>County</em>
+   </td>
+   <td><em>In WOF pre-Karmashapes</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Sub-district</em>
+   </td>
+   <td><em>N/A</em>
+   </td>
+   <td><em>Speculative new WOF placetype, a followup to the Karmashapes import</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Village</em>
+   </td>
+   <td><em>Localadmin and locality</em>
+   </td>
+   <td><em>Significant WOF upgrade post-Karmashapes</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Urban</em>
+   </td>
+   <td><em>Localadmin and locality (sometimes coterminous with county)</em>
+   </td>
+   <td><em>In WOF pre-Karmashapes</em>
+   </td>
+  </tr>
+</table>
+
+
+Additional localization work was done in WOF once Karmashapes geometries and properties were imported. Hindi translations were added to any locality record with a population of 50,000 or more and additional localizations were added based on regional language patterns. WOF maintains official and spoken language by region, so each locality’s parent region was evaluated to prompt additional relevant localizations to match local expectations and supplement the Hindi localizations. In total, name translations were added to over 17,000 records over what was provided by Karmashapes.
+
+Using openly-licensed data at each step of the build process allows Karmashapes to be released under a simple CC-BY license and, in turn, allows Who’s On First to import the full Karmashapes dataset. Without openly licensed data, this build would not have been possible.
+
+# Get the Data
+
+Karmashapes and Who’s On First data are available in several file formats and through various endpoints. You can access and browse data here:
+
+* [Browse](https://spelunker.whosonfirst.org/placetypes/locality/?&iso=in) Who’s On First records in the Spelunker
+* [Download](https://geocode.earth/data/whosonfirst/#IN) Who’s On First data in Shapefile and SQLite formats, via Geocode.Earth: 
+* [Learn](https://whosonfirst.org/blog/2023/05/31/shapefiles/) more about downloading Who’s On First data in various formats
+* Want the raw Karmashapes files? Full Karmashapes downloads are [available](https://karmashapes.wordpress.com/) on Justin’s blog. 
```

Comment:
Link all the way into the blog post: 

- https://karmashapes.wordpress.com/2023/03/05/karmashapes-v0/

As it otherwise lands on a confusing home page requiring another click in.